### DISCUSSION TITLE
Child components re-renders every time I answer a call

### DISCUSSION THREAD

**Discussion by crispinmOHQ** [twilio-professional-services/flex-project-template#616] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/804d8bae-fe90-47bf-8189-fb831220b256)

My Components renders fine. However once, I answer a call, updates happen and It unmounts and mounts my components making it losing its focus and its state as well. It stops doing the rerendering after updating endConferenceOnExit. I believe this is the out of the box of plugin so I do not have control of it. Is there a way for me to control those updates so my components would not rerender?

Thank you

**Comment by dremin** [comment]
Hi @crispinmOHQ! Could you please provide some more detail as to how and where your component is being mounted?

**Reply by crispinmOHQ** [reply, marked as answer]
All good now @dremin I realised I'm mounting my components one by one in a switch statement.
